README.md

```diff
@@ -21,83 +21,83 @@ Sensitive Tree.
+contained there and link to other online resources where more information
 can be obtained.
 
 Links to Wikidata, Antlr definitions, E(BNF) files, example files, will
-be integral components of the definitions here.
+be integral components of the definitions there.
 
 Terminology used to describe language components will be consistent
-across languages where-ever possible, and defer to terminology used
-in academic literature or study guides, to make these definitions more
+across languages wherever possible, and defer to terminology used
+in academic literature or study guides to make these definitions more
 accessible and useful to students of language theory.
 
 ## Stages
 
-1. Organically grow a human readable fact based database of any syntax,
-   stored in YAML files, covering any language from large and complicated
-   programming languages down to strings like a URL, especially focusing
-   on style description which describe a subset of a language.
+1. Organically grow a human-readable fact-based database of any syntax --
+   stored in YAML files -- covering any language, from large and complicated
+   programming languages down to strings like URLs, especially focusing
+   on style descriptions which describe a subset of a language.
 
 2. Create programs to load these definitions and convert input files
-   into a universal AST, primarily for building a test suite to verify the
-   language definitions are able to parse files at useful levels of detail,
-   again focusing on style-defined subsets of languages which are easier
-   and also more useful.
+   into a universal AST, primarily for building a test suite to verify that
+   the language definitions are able to parse files at useful levels of
+   detail. Again, focusing on style-defined subsets of languages, which are
+   easier and also more useful.
 
-   These programs may use existing parsers, by converting the coAST
+   These programs may use existing parsers by converting the coAST
    definitions into metasyntax used by other parsing toolkits, such as BNF
    and derivatives, Antlr .g4, and augeas.
 
 3. Standardise the definition schema once a sufficiently large number
-   of language definitions have been adequately verified to determine
-   the schema is able to usefully describe most concepts found in
-   commonly used grammars.
+   of language definitions have been verified to determine that
+   the schema can usefully describe most concepts found in
+   commonly-used grammars.
 
 ## Phases
 
-These phases will be overlapping slightly.
+These phases will be slightly overlapping.
 
 ### Phase 1: Replace coala language definitions
 
 The language definitions found at
 https://github.com/coala/coala/tree/master/coalib/bearlib/languages
-will be manually added as language definitions here, growing the schema
-as necessary.  Once the import of facts is complete, a generator will
+will be manually added as language definitions, growing the schema
```

Comment:
I think we can keep the `here`, no need to remove it

Reply:
Going off of your comment above, I'd agree.